source/features/clean-repo-sidebar.tsx

```diff
@@ -53,6 +53,13 @@ async function hideEmptyMeta(): Promise<void> {
 	}
 }
 
+async function moveReportLink(): Promise<void> {
+	await domLoaded;
+
+	const reportLink = select('.Layout-sidebar a[href^="/contact/report-content"]')!.parentElement!;
```

Comment:
Not sure if this should use `:has`

Reply:
It's fine as is for now, especially because it wouldn't shorten the code much